### PR TITLE
Back Button and Card Changes

### PR DESCRIPTION
**Pull request by dchicasduena**
Added a back button to the main game and changed the look of the cards

### CODE REVIEW

File: screens/GameScreen.js [hunk 1]
```diff
@@ -78,16 +78,16 @@ const GameScreen = ({navigation}) => {
        <>
        </> 
       ) : (
-      <View className={`${bgColor} flex items-center h-full justify-center`}>
+      <TouchableOpacity onPress={handleNext} className={`${bgColor} flex items-center h-full justify-center`} style={styles.card}>
```
Comment (malikrahey) [inline screens/GameScreen.js:88]: do this with tailwind instead, makes it cleaner

File: styles.js [hunk 2]
```diff
@@ -10,6 +10,11 @@ const styles = StyleSheet.create({
     width: 322,
     height: 310,
   },
+
+  card: {
+    borderBottomLeftRadius: 20,
+    borderBottomRightRadius: 20,
+  }
 });
```
Comment (malikrahey) [inline styles.js]: 🙊

**Review by malikrahey — CHANGES_REQUESTED** [review]

**Comment by dchicasduena** [comment]
changed to tailwind

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -21,22 +21,26 @@ const HomeScreen = ({navigation}) => {
     
       <ScrollView>
 
-      <View className="flex items-center justify-evenly h-screen">
+      <View className="flex items-center justify-center h-screen">
         
       <Image style={styles.logo} source={logo}></Image>
```
Comment (malikrahey) [inline screens/HomeScreen.js:26]: tailwind

File: screens/TimerScreen.js [hunk 4]
```diff
@@ -58,6 +59,12 @@ const TimerScreen = ({ navigation }) => {
...
 
+    const handleBack = () => {
+        ScreenOrientation.unlockAsync(ScreenOrientation.OrientationLock.PORTRAIT);
+        navigation.navigate("Home")
+        return;
+        }
+
     const renderTime = ({ remainingTime }) => {
         if (remainingTime === 0) {
           return <Text style={{ fontSize: '80em' }} className="text-center font-bold">Drink!</Text>;
```
Comment (malikrahey) [inline screens/TimerScreen.js:70]: tailwind, only reason to ever use the styles class is for the AndroidSafeArea really cause we can't do that easily with tailwind

Comment (malikrahey) [reply screens/TimerScreen.js:70]: you can do text-[80em] if none of the other presets work for it too

**Review by malikrahey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**